### DISCUSSION TITLE
How to trigger a method from key

### DISCUSSION THREAD

**Discussion by sqllyw** [alpinejs/alpine#1153] [1. Help] [answered]
Hi,
here is a simple code, click 'search', it calls search_proc, and I'd like to press 'enter' to do the same, but it does not:

https://codepen.io/sqllyw/pen/jOVKGXy?editors=1111

what I'm missing here? Thanks

**Comment by KevinBatdorf** [comment, marked as answer]
You used `keydown.cmd.enter` which requires you press command + enter.

But probably better like this:
```html
<form @submit.prevent='search_proc()'>
  <input type="text" x-model="search">
  <button>search</button>
</form>
```

**Reply by sqllyw** [reply]
right, removing cmd works, will try that better way, thanks
